help/sources/tutorials/ui/create/analytics/rainfocus-experience.md

```diff
@@ -0,0 +1,202 @@
+| transmissionId            | String   |                1680309557133001YHhz               |                Unique identifier for data push               |
+| eventType                 | String   |                  SessionScheduled                 | Name of Attendee Experience Event (see Push Type Dictionary) |
+| timestamp                 | DateTime |              2023-04-01T00:41:57.000Z             |                    Timestamp of data push                    |
+| event.name                | String   |                 Adobe Summit 2023                 |     Name of the event in which a transmission took place.    |
+| exhibitor.exhibitorId     | String   |                1680309557133001YHhz               |             RainFocus Identifier of the exhibitor            |
+| exhibitor.externalId      | String   |                1666809514105001lSJN               |           Identifier for exhibitor in client system          |
+| exhibitor.name            | String   |                        IBM                        |                     Name of the exhibitor                    |
+| lead.leadId               | String   |                1666809456617001wyPj               |               RainFocus Identifier of the lead               |
+| lead.note                 | String   |                                                   |                                                              |
+| session.sessionId         | String   |                1666809373585001t4aV               |             RainFocus Identifier for the session             |
+| session.externalId        | String   |                1666809456617001wyPj               |            Identifier for session in client system           |
+| session.code              | String   |                        GS3                        |                        Code of session                       |
+| session.title             | String   |                Inspiration Keynote                |                       Title of session                       |
+| session.length            | Int      |                         90                        |                       Length of session                      |
+| sessiontime.sessiontimeId | String   |                1673033149739001OJLZ               |           RainFocus Identifier for the session time          |
+| sessiontime.startTime     | String   |                2023-03-22 10:00:00                |                      Session start time                      |
+| sessiontime.endTime       | String   |                2023-03-22 10:00:00                |                       Session end time                       |
+| sessiontime.room          | String   |                        B32                        |                       Room for session                       |
+
+To use the sample Schema from RainFocus, you may take one of the the following steps:
```

Comment:
Maybe we can make this a little more clear, something like: "To create your schema for RainFocus data, read the following documentation for steps on how to create a schema using APIs or the UI."